### PR TITLE
Сделал дз 18_19

### PR DESCRIPTION
**Pull request by Vlaadkaaaa**
https://disk.yandex.ru/i/Lb954hOi_mRlkw

### CODE REVIEW

File: MiniApp/MiniApp/VC/ForYouViewController.swift [hunk 1]
```diff
@@ -7,12 +7,223 @@
 
 import UIKit
 /// Для вас
-final class ForYouViewController: UIViewController {
-
+final class ForYouViewController: UIViewController, UINavigationControllerDelegate {
```
Comment (AlexanderKokh) [inline MiniApp/MiniApp/VC/ForYouViewController.swift:10]:
```suggestion
final class ForYouViewController: UIViewController {
```
extension

File: MiniApp/MiniApp/VC/ForYouViewController.swift [hunk 2]
```diff
@@ -7,12 +7,223 @@
...
+        view.layer.shadowRadius = 3.0
+        view.layer.shadowOpacity = 0.2
+        view.layer.cornerRadius = 10
+        view.addSubview(addLineView(frame: CGRect(x: 0, y: 85, width: 360, height: 2)))
+        
+        let deliveryItems = [
+            itemImageView, orderStatusLabel, deliveryItemCountLabel,
+            deliveryProgressView, deliveryStatusOneLabel, deliveryStatusTwoLabel, deliveryStatusThreeLabel]
+        
+        deliveryItems.forEach { view.addSubview($0)}
```
Comment (AlexanderKokh) [inline MiniApp/MiniApp/VC/ForYouViewController.swift:78]:
```suggestion
        deliveryItems.forEach { view.addSubview($0) }
```

File: MiniApp/MiniApp/VC/WebViewController.swift [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+    
+    private func loadWebSite(urlString: String) {
+        guard let url = URL(string: urlString) else { return }
+        let request = URLRequest(url: url)
+        webView.load(request)
+    }
+    
+    private func checkLoadWebSite() {
+        if !openDoc.isEmpty {
+            guard let request = Bundle.main.url(forResource: openDoc, withExtension: "pdf") else { return }
```
Comment (AlexanderKokh) [inline MiniApp/MiniApp/VC/WebViewController.swift:128]:
```suggestion
            guard let request = Bundle.main.url(forResource: openDoc, withExtension: "") else { return }
```
Constants

File: MiniApp/MiniApp/VC/WebViewController.swift [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+            webView.load(URLRequest(url: request))
+        } else {
+            switch itemName {
```
Comment (AlexanderKokh) [inline MiniApp/MiniApp/VC/WebViewController.swift:131]:
```suggestion
                switch itemName {
            case Constants.itemOneName: 
                loadWebSite(urlString: Constants.urlCase)
            case Constants.itemTwoName: 
                loadWebSite(urlString: Constants.urlWatch)
            case Constants.itemThreeName: 
                loadWebSite(urlString: Constants.urlReStore)
            case Constants.itemFourName: 
                loadWebSite(urlString: Constants.urliPhone)
            default: 
                loadWebSite(urlString: Constants.urlReStore)
            }
```

File: MiniApp/MiniApp/VC/WebViewController.swift [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+        case 2:
+            webView.reload()
+            createTimerForProgressView()
+        default:
+            break
+        }
+    }
+    private func setupUI() {
+        view.backgroundColor = .white
+        let viewElements: [UIView] = [webView, toolBar
```
Comment (AlexanderKokh) [inline MiniApp/MiniApp/VC/WebViewController.swift:160]:
```suggestion
        let viewElements: [UIView] = [webView, toolBar]
```

**Comment by Vlaadkaaaa** [comment]
https://user-images.githubusercontent.com/102802846/195385336-f84503b6-7a2c-48d3-8b02-0bc142664752.mp4

**Comment by Vlaadkaaaa** [comment]
https://user-images.githubusercontent.com/102802846/195588046-dfbc857c-7762-4ff3-92f3-e2abb4167448.mp4

**Review by AlexanderKokh — APPROVED** [review]

**Pull request merged**